### PR TITLE
Update rubocop

### LINKED ISSUE #458: Update rubocop
Github has helpfully commented that there is a security vulnerability in the version of rubocop that Pulmap uses and it should be updated.

screenshot 2017-11-21 12 21 44

### PR DESCRIPTION
**Pull request by eliotjordan**
Updates rubocop to 0.51.0. Closes #458

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -43,44 +43,44 @@ gem 'jbuilder', '~> 2.5'
...
   gem 'capybara'
-  gem 'poltergeist'
-  gem 'factory_girl_rails'
   gem 'database_cleaner'
+  gem 'factory_girl_rails'
+  gem 'poltergeist'
   gem 'rails-controller-testing'
-  gem 'rubocop', '~> 0.42.0', require: false
-  gem 'rubocop-rspec', '~> 1.6.0', require: false
+  gem 'rubocop', '~> 0.5', require: false
```
Comment (escowles) [inline Gemfile]: This should be `0.51.0` right?

**Comment by escowles** [comment]
Other than one thing in the Gemfile, this looks good to me.

File: Gemfile (hunk 1 above)
Comment (eliotjordan) [reply Gemfile]: Yes it should.

**Comment by eliotjordan** [comment]
Gemfile issue fixed.

**Pull request merged**